src/view.rs

```diff
@@ -362,8 +362,19 @@ fn process_keyboard_event(
             BackendCommand::Write(text.as_bytes().to_vec()),
         ),
         egui::Event::Copy => {
-            let content = backend.selectable_content();
-            InputAction::WriteToClipboard(content)
+            #[cfg(not(any(target_os = "ios", target_os = "macos")))]
+            if backend.is_selected_mode() {
```

Comment:
```suggestion
            if modifiers.contains(Modifiers::COMMAND | Modifiers::SHIFT) {
```

I think more effective way is do the same as in other terminal emulators in `linux/windows` and allow only `shift+ctrl+c` for copy and `shift+ctrl+v` for paste events.